neqo-transport/src/tracking.rs

```diff
@@ -396,9 +417,15 @@ impl RecvdPackets {
             unacknowledged_count: 0,
             unacknowledged_tolerance: DEFAULT_ACK_PACKET_TOLERANCE,
             ignore_order: false,
+            ecn_count: EcnCount::default(),
         }
     }
 
+    /// Increase the ECN count for the mark given by `ecn` by `1`.
+    pub fn inc_ecn_count(&mut self, ecn: IpTosEcn) {
+        self.ecn_count[ecn] += 1;
```

Comment:
If you have a new type for `EcnCount` you could implement `std::ops::Add<IpTosEcn>` for this.  Just a thought.

Reply:
I'd need to make `RecvdPackets::ecn_count` public though, no?

Reply:
Since you did implement `Add`:

```suggestion
        self.ecn_count += ecn;
```
:)